src/VisualStudio/Xaml/Impl/Implementation/LanguageClient/XamlInProcLanguageClient.cs

```diff
@@ -46,6 +46,7 @@ internal class XamlInProcLanguageClient : AbstractInProcLanguageClient
 
         /// <summary>
         /// Gets the name of the language client (displayed in yellow bars).
+        /// When updating the string of Name, please make sure to update the same string in Microsoft.VisualStudio.LanguageServer.Client.ExperimentalSnippetSupport.AllowList
```

Comment:
@NTaylorMullen wanted to check, is snippet support being expanded outside of razor

Reply:
Yeah we have already chatted with Taylor about adding this support in XAML.

Reply:
Yup we good!